TipCatDotNet.Api/Services/Payments/TransactionService.cs

```diff
@@ -28,13 +30,20 @@ public TransactionService(AetherDbContext context)
     public async Task<Result> Add(PaymentIntent paymentIntent, string? message, CancellationToken cancellationToken = default)
     {
         var memberId = int.Parse(paymentIntent.Metadata["MemberId"]);
+
+        var facilityId = await _context.Members
+            .Where(m => m.Id == memberId)
+            .Select(m => m.FacilityId)
+            .SingleOrDefaultAsync();
+
         var now = DateTime.UtcNow;
 
         var newTransaction = new Transaction
         {
             Amount = MoneyConverting.ToFractionalUnits(paymentIntent),
             Currency = paymentIntent.Currency,
             MemberId = memberId,
+            FacilityId = facilityId ?? 0,
```

Comment:
I think, this situation should be impossible by design. Better log and fail here